package.json

```diff
@@ -2248,6 +2248,11 @@
 					"command": "pr.collapseAllComments",
 					"group": "collapse@0",
 					"when": "commentController =~ /^github-(browse|review)/"
+				},
+				{
+					"command": "pr.refreshComments",
+					"group": "refresh@0",
```

Comment:
The order here seems wrong. I would keep this before the collapse action. 

<img width="179" alt="image" src="https://github.com/microsoft/vscode-pull-request-github/assets/38270282/5b4d1d13-aadf-4752-a108-525cfe2a87d3">

Reply:
The order is determined by the `group`, not but the position in the `package.json`.

Reply:
Sorry for the delay. I've fixed it now.